content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -473,3 +473,15 @@ Now that you know how to use checkpoints in DVC, you'll be able to resume
 training from different checkpoints to try out new hyperparameters or code and
 you'll be able to track all of the changes you make while trying to create the
 best possible model.
+
+## checkpoint backup
+
+`DVC_EXP_AUTO_PUSH` env var is intended to help backup the experiments results
+which helps to recover from interruptions that happened in the middle of a model
```

Comment:
I think we don't need a new section. We can put all this info at the end of[ **Running experiments**](https://dvc.org/doc/user-guide/experiment-management/checkpoints#running-experiments) (before **Viewing checkpoints**).

Reply:
I see it was moved but we still have the header. @casperdcl maybe you can summarize so it's all just one paragraph or a bit more at the end of **Running experiments**.

Reply:
ok reworded. I think it's a good idea to keep a header for the `#id` backlink which people can use to reference